ndk/src/event.rs

```diff
@@ -47,18 +47,30 @@ pub enum Source {
     Any = ffi::AINPUT_SOURCE_ANY,
 }
 
-/// An enum representing the class of an [`InputEvent`] source.
-///
-/// See [the NDK docs](https://developer.android.com/ndk/reference/group/input#anonymous-enum-35)
```

Comment:
I've stripped off this doc because it's been renumbered upstream and points nowhere.